firmwares/aeotec/ZWA024-A.json

```diff
@@ -0,0 +1,31 @@
+{
+	"devices": [
+		{
+			"brand": "aeotec",
+			"model": "ZWA024-A",
+			"manufacturerId": "0x0371",
+			"productType": "0x0102",
+			"productId": "0x0018",
+			"firmwareVersion": {
+				"min": "0.0",
+				"max": "1.5"
+			}

```

Comment:
Limiting the firmware range is redundant when you're adding a condition to the upgrades. I'd probably remove this and keep the condition on each upgrade so all updates can be defined in the same file.

Reply:
I'll remove this, i'll use conditions since i can add a check for "productType" instead since that'll help ensure its with the correct region.